code_schemes/gender.json

```diff
@@ -8,23 +8,32 @@
       "CodeType": "Normal",
       "DisplayText": "male",
       "NumericValue": 1,
+      "StringValue": "male",
       "Shortcut": "m",
-      "VisibleInCoda": true
+      "VisibleInCoda": true,
+      "MatchValues" : [
+        "male"
+      ]
     },
     {
       "CodeID": "code-86a4602c",
       "CodeType": "Normal",
       "DisplayText": "female",
       "NumericValue": 2,
+      "StringValue" : "female",
       "Shortcut": "f",
-      "VisibleInCoda": true
+      "VisibleInCoda": true,
+      "MatchValues" : [
+        "female"
+      ]
     },
     {
       "CodeID": "code-NA-f93d3eb7",
       "CodeType": "Control",
       "ControlCode": "NA",
       "DisplayText": "NA (missing)",
       "NumericValue": -10,
+      "StringValue": "NA",
```

Comment:
Why don't the control codes have match values?

Reply:
It seems like a fairly substantial layering violation, and additional coupling for string values. 

If a cleaner can't clean "gawreoihgor", I'm not sure it can know whether it's an NA, WS, NIC, that seems like it's context dependent.

Anything else further in the pipeline, it should be using the parsed schemes and matching the control codes?